packages/appstore/src/components/wallet-header/wallet-header-title.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { TWalletSvgCurrency } from 'Types';
+
+type TWalletHeaderTitle = {
+    is_demo: boolean;
+    currency: TWalletSvgCurrency;
+    jurisdiction: 'svg' | 'malta';
+};
+
+const WalletHeaderTitle = React.memo(({ is_demo, currency, jurisdiction }: TWalletHeaderTitle) => {
+    const title = is_demo ? (
+        <Text weight='bold' size='sm'>
+            <Localize
+                i18n_default_text='Demo {{currency}} Wallet'
+                values={{
+                    currency,
+                }}
+            />
+        </Text>
+    ) : (
+        <Text weight='bold' size='sm'>
+            <Localize
+                i18n_default_text='{{currency}} Wallet'
+                values={{
+                    currency,
+                }}
+            />
+        </Text>
+    );
```

Comment:
```suggestion
    const title = (
        <Text weight='bold' size='sm'>
    <Localize
      i18n_default_text={`${is_demo ? 'Demo ' : ''}{{currency}} Wallet`}
      values={{ currency }}
    />
  </Text>
    );
```

Reply:
@thisyahlen-deriv using variables in Localize is not allowed. please check this link (https://deriv-group.slack.com/archives/C3AHMSM3P/p1677553396124379)

Reply:
I believe its `localize` is not advised. Its fine for <Localize/> component

Reply:
@thisyahlen-deriv 
<img width="1757" alt="Screenshot 2023-05-02 at 9 34 55 AM" src="https://user-images.githubusercontent.com/120570511/235596821-35589570-a160-4dde-bea6-c3be101a9152.png">
I suppose it's related to `Localize` too